pkg/cli/flags.go

```diff
@@ -51,6 +51,7 @@ var serverSQLAddr, serverSQLPort string
 var serverSQLAdvertiseAddr, serverSQLAdvertisePort string
 var serverHTTPAddr, serverHTTPPort string
 var localityAdvertiseHosts localityList
+var clockDevice string
```

Comment:
This needs to be reset to its default value in `context.go`. However see my next comment.

Reply:
removed